components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```diff
@@ -843,132 +841,55 @@ def read_and_update_settings_json(settings_file_path: pathlib.Path, updates: Dic
     return settings_object
 
 
-def start_webui(instance_id: str, clp_config: CLPConfig, mounts: CLPDockerMounts):
-    component_name = WEBUI_COMPONENT_NAME
-    logger.info(f"Starting {component_name}...")
-
-    container_name = f"clp-{component_name}-{instance_id}"
-    if container_exists(container_name):
-        return
-
-    webui_logs_dir = clp_config.logs_directory / component_name
-    container_webui_dir = CONTAINER_CLP_HOME / "var" / "www" / "webui"
-    node_path = str(container_webui_dir / "programs" / "server" / "npm" / "node_modules")
-    settings_json_path = get_clp_home() / "var" / "www" / "webui" / "settings.json"
-
-    validate_webui_config(clp_config, webui_logs_dir, settings_json_path)
-
-    # Create directories
-    webui_logs_dir.mkdir(exist_ok=True, parents=True)
-
-    container_webui_logs_dir = pathlib.Path("/") / "var" / "log" / component_name
-
-    # Read and update settings.json
-    clp_db_connection_params = clp_config.database.get_clp_connection_params_and_type(True)
-    table_prefix = clp_db_connection_params["table_prefix"]
-    if StorageEngine.CLP_S == clp_config.package.storage_engine:
-        table_prefix = f"{table_prefix}{CLP_DEFAULT_DATASET_NAME}_"
-    meteor_settings_updates = {
-        "private": {
-            "SqlDbHost": clp_config.database.host,
-            "SqlDbPort": clp_config.database.port,
-            "SqlDbName": clp_config.database.name,
-            "SqlDbClpArchivesTableName": f"{table_prefix}{ARCHIVES_TABLE_SUFFIX}",
-            "SqlDbClpFilesTableName": f"{table_prefix}{FILES_TABLE_SUFFIX}",
```

Comment:
Since we're using dataset-specific tables for clp-json, we'll need to pass these into the webui from here, right?

Reply:
gg.. sorry for missing this